packages/extra/monitoring/templates/grafana/grafana.yaml

```diff
@@ -26,11 +26,34 @@ spec:
     security:
       admin_user: user
       admin_password: ${GF_PASSWORD}
+    plugins:
+      allow_loading_unsigned_plugins: "victorialogs-datasource"
   deployment:
     spec:
       replicas: 2
       template:
         spec:
+          initContainers:
+            - name: "load-vm-ds-plugin"
```

Comment:
better to build custom image as there could be no internet access (prepare for airgap)

Reply:
or just use a VolumeSource from the latest kubernetes and use the power of OCI

Reply:
makes sense, I'll add an issue for this https://github.com/aenix-io/cozystack/issues/306